packages/video_player/example/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
```

Comment:
Do we need to bump this here? 

I would probably prefer to keep all the gradle versions in sync for this repo - if we need to upgrade it here, could we move it to a separate PR and bump all the gradle-wrapper.properties files?

Reply:
Alright.. I was surprised to find 4.10 in use, as 4.10.2 has been released soon after. Will reactivate #817 and send asap.

Reply:
I'm fine with going to 4.10.2, but I'd just like to do it for all of them at once so that it's easier to find/replace on the whole repo in the future - and preferably do it as a separate commit so we can track the history and revert if need be.

Reply:
OK. Updated #817 ..